README.md

```diff
@@ -14,3 +14,12 @@ Due to the earliness of the operation, the rootfs and boot partitions are manual
 After the recovery, they are restored to the previous state.
 
 To fetch the log from a recovery, execute after bootup: `sudo systemctl status kano-os-recovery -b`.
+
+## Querying if system APT is stable
+
+Simply running `sudo kano-os-recovery --info` will display any broken packages, and return exit code 0 if recovery is needed.
```

Comment:
By the looks of it, exit 0 happens when a recovery has happened or one wasn't needed.

Reply:
Blah, scratch that - misread :)